repository/src/main/java/org/alfresco/ibatis/SqlSessionMetricsWrapper.java

```diff
@@ -192,6 +193,24 @@ private void reportQueryExecuted(final long startTime, final String queryTypeTag
         }
     }
 
+    @Override
+    public <T> Cursor<T> selectCursor(String s)
+    {
+        return null;
```

Comment:
This requires implementation. See other methods in this class how they delegate to the default implementation of mybatis.

Reply:
Implemented.